playbooks/besman-counterfit-0.0.1-playbook.sh

```diff
@@ -77,6 +85,9 @@ function __besman_execute() {
     done
 
     [[ -z $COUNTERFIT_ATTACKID ]] && __besman_echo_red "Attack Id is not set. Required. Please set it and try again." && return 1
+    [[ ! -f counterfit/targets/$BESMAN_ARTIFACT_NAME.py ]] && __besman_echo_red "$BESMAN_ARTIFACT_NAME.py not copied to targets folder." && return 1
```

Comment:
Provide the complete path everywhere - $HOME/counterfit/counterfit/targets..

Reply:
Changed